<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="coord_point_1t">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Koordinat Titik 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="azimuth_1">
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Azimuth titik 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_titik_1t">
       <property name="text">
        <string>Titik 1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="coord_point_2t">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Koordinat Titik 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="azimuth_2">
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Azimuth titik 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_titik_2t">
       <property name="text">
        <string>Titik 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
